config/routes.rb

```diff
@@ -2,5 +2,8 @@
 
   mount_devise_token_auth_for 'User', at: 'auth'
   resources :books, only: [:index, :show]
-  
+  resources :rent, only: [:list, :create]
```

Comment:
`only: [:index, :create]` 👀 Remember my comment about using REST standards with the Rails convention